sql/moz-fx-data-shared-prod/firefox_ios_derived/app_store_choice_screen_selection_v1/schema.yaml

```
@@ -12,7 +12,8 @@ fields:
   type: STRING
   description: |
     The user’s region code as set in Settings > General > Language & Region. \
-    This may not correspond to the user’s Apple Account or App Store storefront.
+    This may not correspond to the user’s Apple Account or App Store storefront. \
+    Starting 23-06-2025 this field will correspond to the user's App Store storefront region.
```

Looking for consistency with the file above.
```suggestion
    Starting 2025-06-23 this field will correspond to the user's App Store storefront region.
```

---

Yes, you are correct. Not sure what was going on in my head when I swapped the two around 😅 Thanks!